Gems/Atom/Tools/AtomToolsFramework/Code/Source/PreviewRenderer/PreviewRendererCaptureState.h

```diff
@@ -28,11 +28,10 @@ namespace AtomToolsFramework
         //! AZ::TickBus::Handler interface overrides...
         void OnTick(float deltaTime, AZ::ScriptTimePoint time) override;
 
-        //! AZ::Render::FrameCaptureNotificationBus::Handler overrides...
-        void OnCaptureFinished(uint32_t frameCaptureId, AZ::Render::FrameCaptureResult result, const AZStd::string& info) override;
+        //! AZ::Render::FrameCaptureNotificationBus::MultiHandler overrides...
+        void OnFrameCaptureFinished(AZ::Render::FrameCaptureId frameCaptureId, AZ::Render::FrameCaptureResult result, const AZStd::string& info) override;
```

Comment:
You can remove the frame capture ID from the parameter list now that you are connecting to it. Handlers can get the frame capture ID by using
```suggestion
        //! AZ::Render::FrameCaptureNotificationBus::MultiHandler overrides...
        void OnFrameCaptureFinished(AZ::Render::FrameCaptureResult result, const AZStd::string& info) override;
```

const AZ::Render::FrameCaptureId frameCaptureId = *AZ::Render::FrameCaptureNotificationBus::GetCurrentBusId();